website/docs/utilities/popover-primitive/partials/code/how-to-use.md

```diff
@@ -0,0 +1,133 @@
+!!! Warning
+
+This component is intended only for internal Helios use. If you need to use it, please [contact](/about/support) the Design Systems Team.
+!!!
+
+## How to use this component
+
+The `PopoverPrimitive` is a **headless component** that associates a "toggle" element with a "popover" element (both elements act as containers). "Soft" (hover/focus) or "click" event listeners can be assigned to the "toggle", and when triggered they toggle the visibility of the "popover".
+
+When the "popover" is visible, it can be closed in various ways: toggling via the "soft" or "click" events, clicking outside of the popover, or via the `esc` key.
```

Comment:
```suggestion
When the popover is visible, it can be closed in various ways: toggling via the "soft" or click events, clicking outside of the popover, or via the `esc` key.
```

Reply:
I'm leaving "soft" in quotes since a HDS coined term and seems more likely to be misunderstood but it may not be necessary for it either...

Found this reference:
https://writingcenter.uagc.edu/quotation-marks#:~:text=Use%20quotation%20marks%20around%20unusual,been%20introduced%20the%20first%20time.

Normally you only need to use quotes the first time when introducing a technical term but not throughout the entire document.

Reply:
I wonder if we should update this text to use `light dismiss` since that is the technical term for pressing the ESC key or clicking outside to close.